cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/partitioned/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright © 2015-2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/**
+ * Implementation of partitioned datasets in CDAP.
```

Comment:
This package does not contain the partitioned datasets, but currently contains the classes which are consuming the partitions. However this can change in future. Should we say here `Implementation of the classes which uses partitioned datasets(such as consumers of the partitions) in CDAP.`?